src/components/App/MainMenu.js

```diff
@@ -218,7 +218,13 @@ class MainMenuView extends React.Component {
                     <p>{visualizationType} Analytics</p>
                 </div>
 
-                <Subheader style={style.subHeader}>ENTERPRISES <span className="pull-right"><img style={style.iconSubMenu} src="/icons/icon-enterprise.png" alt="" /></span></Subheader>
+                <Subheader style={style.subHeader}>
+                    ENTERPRISES
+                    <span className="pull-right">
+                      <img style={style.iconSubMenu} src={`${process.env.PUBLIC_URL}/icons/icon-enterprise.png`} alt="" />
```

Comment:
Isnt this line the only change that needs to change?

Reply:
Agreed, and it is same as previous, but used the latest format as per ECMA 6 and have better readbility. A bit of refactoring along with fixing the issue.

For your reference: https://developers.google.com/web/updates/2015/01/ES6-Template-Strings